coreax/kernel.py

```diff
@@ -335,9 +335,14 @@ def divergence_x_grad_y_elementwise(self, x: ArrayLike, y: ArrayLike) -> Array:
 
 
 class SquaredExponentialKernel(Kernel):
-    """
+    r"""
     Define a squared exponential kernel.
 
+    The squared exponential kernel is defined as
+    :math:`k: \mathbb{R}^d\times \mathbb{R}^d \to \mathbb{R}`,
+    :math:`k(x, y) = output_scale \exp(\frac{||x-y||^2}{2 length_scale^2})` where
```

Comment:
The `output_scale` and `length_scale` either need their own symbols, or wrapping in a \text block. I'd prefer a symbol but either is fine. This holds for the other docstrings as well.